src/applications/static-pages/i18Select/I18Select.js

```diff
@@ -22,7 +22,7 @@ const I18Select = ({ baseUrls, languageCode }) => {
           return (
             <span key={i}>
               <a
-                className={`vads-u-font-size--base vads-u-font-family--sans vads-u-padding-bottom-0p5 ${
+                className={`i18-select vads-u-font-family--sans vads-u-padding-bottom-0p5 ${
```

Comment:
The base font size class `vads-u-font-size--base` was forcing this content to display as 16px. I added a custom class here so that we could apply `1.06rem` instead.